### PR TITLE
Week 9 blog post

### PR DESCRIPTION
**Pull request by m-agour**
My blog post for last week.

### CODE REVIEW

**Review by xtanion — COMMENTED** [review]
Nice blog @m-agour, It will be really helpful to have hierarchical transformation in actors. There are no typos in this blog, It looks good to me.
Thanks

File: docs/source/posts/2022/2022-08-16-week-9-mohamed.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.. post:: August 16 2022
+   :author: Mohamed Abouagour
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- Made two tutorials that show two approaches on how to animate primitives of the same FURY actor using the ``Timeline``.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-08-16-week-9-mohamed.rst:13]: Can you please add a link to this PR?

**Review by ganimtron-10 — COMMENTED** [review]
Hello @m-agour ,
Short and nice blog!

Just a suggestion,
If some points can be elaborate more can you please try to do so?
The current version gives a nice overview, but some people may love to read more about it.

Thanks!

**Review by ganimtron-10 — COMMENTED** [review]
Hello @m-agour ,
Short and nice blog!

Just a suggestion,
If some points can be elaborate more can you please try to do so?
The current version gives a nice overview, but some people may love to read more about it.

Thanks!

File: docs/source/posts/2022/2022-08-16-week-9-mohamed.rst (hunk 1 above)
Comment (skoudoro) [reply docs/source/posts/2022/2022-08-16-week-9-mohamed.rst:13]: +1 for the link.

File: docs/source/posts/2022/2022-08-16-week-9-mohamed.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+What is coming up next week?
+----------------------------
+- Alter the ``Timeline`` to use matrices instead of setting values directly to allow hierarchical transformation.
+
+- Improve the API of the ``PartialActor`` to act almost like a normal actor.
+
+Did you get stuck anywhere?
+---------------------------
+I had some issues but didn't get stuck.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-08-16-week-9-mohamed.rst:31]: some issues? can you explain a bit? thanks!

**Review by skoudoro — COMMENTED** [review]
this PR is almost ready to go. Can you address @ganimtron-10 and my comments? 

Thanks!

**Comment by skoudoro** [comment]
thank you for the update @m-agour , merging

**Pull request merged**